### PR TITLE
initialize ROB with correct metric interval

### LINKED ISSUE #1196: reorderBuffer.interval is not set correctly
the reorderBuffer is created with `NewReorderBuffer(reorderWindow, ret.SecondsPerPoint)`.  where `ret` is a `conf.Retention`.   Unlike in graphite, the default storageSchemas dont explicitly set the secodsPerPoint of every series. Instead, we map retention policies to a range of metric intervals.  We then use the "interval" property sent with each MetricData payload.  

With a default config, the secondsPerPoint is set to "1s" and this is what is being passed when creating the reorderBuffer.

### PR DESCRIPTION
**Pull request by replay**
Not tested yet

fixes #1196

### CODE REVIEW

File: mdata/aggregator.go [hunk 1]
```diff
@@ -43,31 +43,31 @@ func NewAggregator(store Store, cachePusher cache.CachePusher, key schema.AMKey,
 		case conf.Avg:
 			if aggregator.sumMetric == nil {
 				key.Archive = schema.NewArchive(schema.Sum, span)
-				aggregator.sumMetric = NewAggMetric(store, cachePusher, key, conf.Retentions{ret}, 0, nil, dropFirstChunk)
+				aggregator.sumMetric = NewAggMetric(store, cachePusher, key, conf.Retentions{ret}, 0, 0, nil, dropFirstChunk)
```
Comment (Dieterbe) [inline mdata/aggregator.go]: why would we deliberately pass interval=0 and not the correct interval (`span`) ?  even if we don't end up using the value in NewAggMetric() from these calls, this makes the code brittle and likely to break once we start using it for something else.

**Comment by Dieterbe** [comment]
I fixed up this PR. looks good to me now

**Comment by woodsaj** [comment]
LGTM.  I added a quick fix to address #1201 as well.

**Comment by Dieterbe** [comment]
moved that commit to #1276

**Pull request merged**